Tests/Foundation/TestJSONEncoder.swift

```diff
@@ -237,72 +237,36 @@ class TestJSONEncoder : XCTestCase {
 
     func test_encodingOutputFormattingSortedKeys() throws {
         let expectedJSON = try XCTUnwrap("""
-        {"2":"2","7":"7","25":"25","ａｌｉｃｅ":"ａｌｉｃｅ","bob":"bob","Charlie":"Charlie","中国":"中国","日本":"日本","韓国":"韓国"}
```

Comment:
The sorting order changed in recent JSONEncoder implementations to be UTF8 byte based instead of a localized sort.